### PR TITLE
Ensure that "Data Infrastructure / Historian" page is up to standard

### PR DESCRIPTION
**Pull request by Ayato77**
# Description
- Information about the data model is updated
- SQL query example is added
- Links are updated

## Related issues

closes united-manufacturing-hub/MgmtIssues/issues/1059

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The Historian / Data Storage feature of the United Manufacturing Hub allows you
 
 ### Store and analyze data
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md:20]: we need to mention the new datamodel

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md [hunk 2]
```diff
@@ -41,15 +41,15 @@ More information about the exact analytics functionalities can be found in the [
...
+- Only data in `processValue` topics are saved automatically. Data in topics like `ia/raw` are not. Data send to other messages in the [UMH datamodel](/docs/datamodel/) are stored slightly different and can be retrieved via Grafana as well. See also [Analytics feature](/docs/features/datainfrastructure/analytics/).
 - After storing a couple of millions messages, you should consider [compressing the messages or establishing retention policies](/docs/production-guide/administration/reduce-database-size/).
-- At the moment, extensive queries can only be done in your own code by leveraging the API in [factoryinsight](/docs/architecture/microservices/core/factoryinsight/), or processing the data in the [Unified Namespace](/docs/features/unified-namespace/).
+- At the moment, extensive queries can only be done in your own code by leveraging the API in [factoryinsight](/docs/architecture/microservices/core/factoryinsight/), or processing the data in the [Unified Namespace](/docs/features/datainfrastructure/unified-namespace/).
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md:52]: not only via API, but also via SQL queriesi n the new datamodel. Also mention an example query (it should not be a tutorial, but as the simple SQL queries are part of our feature set, we should also show how easy it is)

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ The Historian / Data Storage feature in the United Manufacturing Hub provides re
...
+- Store data from the topics with the basic format `.../_historian` or `.../_historian/<tagGroup1>/<tagGroup2>/.../<tagGroupX>/<tagname>` in the Unified Namespace. [Data can be sent to the Unified Namespace from various sources](/docs/features/datainfrastructure/unified-namespace/), allowing you to store tags from your PLC and production lines reliably. Optionally, you can use tag groups to manage a large number of tags and reduce the system load. [This article](https://learn.umh.app/lesson/data-modeling-in-the-unified-namespace-mqtt-kafka/) assists you in learning data modeling in the Unified Namespace.
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md:22]: We should link here to the upcoming data model article by @Scarjit and not to the learn article.

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md (hunk 3 above)
Comment (Ayato77) [reply umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md]: Links are added/updated

**Review by DaruZero — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md [hunk 4]
```diff
@@ -3,58 +3,126 @@ title: Historian / Data Storage
...
+Historian is just another term for a data storage system, designed specifically
+for time-series data in manufacturing.
 
 ## When should I use it?
 
-If you want to reliably store data from your shop floor that is not designed to fulfill any legal purposes, such as GxP, then the United Manufacturing Hub's Historian feature is ideal. Open-Source databases such as TimescaleDB [are superior to traditional historians](https://learn.umh.app/blog/historians-vs-open-source-databases-which-is-better/) in terms of [reliability, scalability and maintainability](https://learn.umh.app/blog/comparing-mqtt-brokers-for-the-industrial-iot/#three-main-requirements-for-your-it-ot-architecture), but can be challenging to use for the OT engineer. The United Manufacturing Hub fills this usability gap, allowing OT engineers to easily ingest, process, and store data permanently in an Open-Source database.
+If you want to reliably store data from your shop floor that is not designed to
+fulfill any legal purposes, such as GxP, then the United Manufacturing Hub's
+Historian feature is ideal. Open-Source databases such as TimescaleDB are
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md:20]: don't say "ideal". makes it sound like marketing

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md [hunk 5]
```diff
@@ -3,58 +3,126 @@ title: Historian / Data Storage
...
+[superior to traditional historians](https://learn.umh.app/blog/historians-vs-open-source-databases-which-is-better/)
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md:21]: *in our opinion

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md [hunk 6]
```diff
@@ -16,8 +16,8 @@ for time-series data in manufacturing.
 ## When should I use it?
 
 If you want to reliably store data from your shop floor that is not designed to
-fulfill any legal purposes, such as GxP, then the United Manufacturing Hub's
-Historian feature is preferred. In our opinion, open-Source databases such as TimescaleDB are
+fulfill any legal purposes, such as GxP, we recommend you to use the United Manufacturing Hub's
+Historian feature. In our opinion, open-Source databases such as TimescaleDB are
```
Comment (DaruZero) [inline umh.docs.umh.app/content/en/docs/features/datainfrastructure/historian.md:20]: Typo in open-Source: should not be capital S

**Review by DaruZero — CHANGES_REQUESTED** [review]

**Pull request merged**